composer.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "multiversx/mx-sdk-php",
+  "description": "MultiversX PHP SDK for interacting with the MultiversX blockchain (in general) and Smart Contracts (in particular).",
+  "scripts": {
+    "test": "vendor/bin/pest",
+    "test-coverage": "vendor/bin/phpunit --coverage-html coverage"
+  },
+  "license": "MIT",
+  "autoload": {
+    "psr-4": {
+      "MultiversX\\": "src"
+    }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "MultiversX\\Tests\\": "tests"
+    }
+  },
+  "authors": [],
+  "require": {
```

Comment:
Using locked versions is not standard practice for defining composer dependencies because it is highly probable for package consumers to face version conflicts given how composer resolves project dependencies:

_Composer (PHP) and NPM (Node.js) handle multiple versions of dependencies differently. Composer installs a single version of a package per project, focusing on dependency compatibility within that project. It resolves dependency conflicts to ensure that only one compatible version of each package is used._

---

I listed the dependencies with info in the PR description

Reply:
Patch version should always be safe to upgrade, no breaking changes expected.